demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -582,6 +581,7 @@ export class DemoMeetingApp
       this.enableUnifiedPlanForChromiumBasedBrowsers = !(document.getElementById(
         'planB'
       ) as HTMLInputElement).checked;
+      this.usePriorityBasedDownlinkPolicy = (document.getElementById('priority-downlink-policy') as HTMLInputElement).checked;
```

Comment:
Do we need to do this? I thought it was already set above.

Reply:
Firefox will save your check boxes.  Since this is default unchecked I added this for convenience.  Though there are probably other places we need to account for this as well.

Reply:
Thank you for doing this!